snippets/nordic-bt-rpc/README.rst

```diff
@@ -3,10 +3,14 @@
 Nordic Bluetooth Low Energy Remote Procedure Call snippet (nordic-bt-rpc)
 #########################################################################
 
+.. contents::
+   :local:
+   :depth: 2
+
 Overview
 ********
 
-This snippet allows you to build Bluetooth applications with Bluetooth stack running on a core dedicated for radio communication.
+This snippet allows you to build Bluetooth® applications with Bluetooth stack running on a core dedicated for radio communication.
```

Comment:
Shouldn't the trademark be in the first instance? In this case the name - line 3?

Reply:
@mia-ko

Reply:
In our DITA-based docs, we don't have trademarks in headings/titles. To keep the consistency with our DITA docs, we are leaving trademarks out of the headings and titles in rst files as well.
Until further notice :)

Reply:
Ok, just wanted to make sure